src/components/AssetsDropZone/index.jsx

```diff
@@ -1,18 +1,13 @@
 import React from 'react';
-import { connect } from 'react-redux';
 import classNames from 'classnames';
 import Dropzone from 'react-dropzone';
 import PropTypes from 'prop-types';
 
 import Button from '@edx/paragon/src/Button';
 import FontAwesomeStyles from 'font-awesome/css/font-awesome.min.css';
-
-import { uploadAssets, uploadExceedMaxSize, uploadExceedMaxCount } from '../../data/actions/assets';
-
 import styles from './AssetsDropZone.scss';
 
-
-export class AssetsDropZone extends React.Component {
+class AssetsDropZone extends React.Component {
```

Comment:
This is just pulling the mapStateToProps, mapDispatchToProps, and connect out of AssetsDropZone/index.jsx into AssetsDropZone/container.jsx; there's no new code